chart/templates/saml-metadata.yaml

```
@@ -13,9 +13,9 @@ spec:
     redirectURL: https://{{ include "gateway.host" . }}/Redirect
     orgName: {{ .Release.Name }}
     orgDisplayName: {{ .Release.Name }}
-    orgURL: example.com
-    contactCompany: example
-    contactGivenName: jeff
-    contactSurname: jefferson
-    contactEmail: jeff@example.local
+    orgURL: https://www.gov.uk/verify
+    contactCompany: GOV.UK Verify
+    contactGivenName: eidas-support
+    contactSurname: eidas-support
+    contactEmail: eidas-support@digital.cabinet-office.gov.uk
```

```suggestion
    contactEmail: idasupport@digital.cabinet-office.gov.uk
```
We should provide the standard Verify support e-mail